bgpd/bgp_nht.c

```diff
@@ -700,9 +704,29 @@ static void bgp_nht_ifp_table_handle(struct bgp *bgp,
 				     struct interface *ifp, bool up)
 {
 	struct bgp_nexthop_cache *bnc;
+	struct nexthop *nhop;
+	bool nhop_found = false;
+	bool nhop_other = false;
 
 	frr_each (bgp_nexthop_cache, table, bnc) {
```

Comment:
`nhop_found = false;` must be in the loop

Reply:
the other variable is neither not needed as the presence of nhop pointer is enough